spark-java/src/main/java/com/baeldung/sparkjava/UserServiceMapImpl.java

```diff
@@ -0,0 +1,64 @@
+package com.baeldung.sparkjava;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class UserServiceMapImpl  implements UserService{
+    private HashMap<String, User> userMap;
+    
+    public UserServiceMapImpl() {
```

Comment:
Use the @Overrides annotation on all the public methods that you are implementing from the interface